environment.yml

```diff
@@ -0,0 +1,367 @@
+name: ased
```

Comment:
Just note that conda export is not useful for making an environment that other people can install, because most of these dependencies need to be under 'pip'. To reproduce, just go onto another machine and try `conda env create -f environment.yml`